Scenes/AudioStreamConductor/AudioStreamConductor.gd

```diff
@@ -42,7 +42,7 @@ func _report_beat():
 		last_beat_lead_up = next_beat
 		emit_signal("beat_lead_up", last_beat_lead_up)
 
-func _physics_process(delta):
+func _physics_process(_delta):
```

Comment:
We had a warning that we weren't using delta and to use _delta instead...not sure if that has any merit.